packages/@aws-cdk/aws-stepfunctions-tasks/test/ecs-tasks.test.ts

```diff
@@ -57,6 +57,7 @@ test('Running a Fargate Task', () => {
 
   // WHEN
   const runTask = new sfn.Task(stack, 'RunFargate', { task: new tasks.RunEcsFargateTask({
+    serviceIntegrationPattern: sfn.ServiceIntegrationPattern.RUN_A_JOB,
```

Comment:
Does synchronous or asynchronous make more sense as a default? What are users most likely to want?